src/Core/src/Handlers/NavigationPage/NavigationViewHandler.Windows.cs

```diff
@@ -1,738 +1,46 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 using Microsoft.Maui.Handlers;
 using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media.Animation;
 
 namespace Microsoft.Maui.Handlers
 {
-	internal partial class NavigationViewHandler :
-		ViewHandler<INavigationView, FrameworkElement>
+	public partial class NavigationViewHandler :
+		ViewHandler<INavigationView, NavigationFrame>
```

Comment:
Can't this be Frame ?

Reply:
https://github.com/dotnet/maui/pull/2551/files#diff-0f779c9331c0bb59305c07a8c6ea0a02d6e0df8a46332931c5f3bb8739f3bf3cR7

Reply:
Right, but we can cast it when we use it , and just have Frame here no ?

Reply:
Check it out now 🔥🔥🔥🔥🔥🔥